include/fiction/io/print_layout.hpp

```diff
@@ -307,6 +316,86 @@ void print_cell_level_layout(std::ostream& os, const Lyt& layout, const bool io_
     // flush stream
     os << std::endl;
 }
+/**
+ * Writes a simplified 2D representation of a SiDB charge layout to an output stream.
+ *
+ * @tparam Lyt Cell-level layout based in SiQAD-coordinates.
+ * @param os Output stream to write into.
+ * @param cds The charge distribution surface of which the charge layout is to be printed.
+ * @param cs_color Flag to utilize color escapes for charge states.
+ */
+template <typename Lyt>
+void print_charge_layout(std::ostream& os, const charge_distribution_surface<Lyt>& cds,
+                         const bool cs_color = false) noexcept
+{
+    // empty layout
+    if (cds.num_cells() == 0ul)
+    {
+        os << "[i] empty layout" << std::endl;
+        return;
+    }
+
+    for (decltype(cds.y()) y_pos = 0; y_pos <= cds.y(); ++y_pos)
+    {
+        for (decltype(cds.x()) x_pos = 0; x_pos <= cds.x(); ++x_pos)
+        {
+            cell<Lyt> c{x_pos, y_pos};
+
+            const fmt::text_style no_color{};
```

Comment:
```suggestion
            static const fmt::text_style no_color{};
```

Reply:
Fixed this in the next commit, though I'm waiting for the tests this time.